Core/Core/Extensions/InstIconExtensions.swift

```diff
@@ -391,6 +391,7 @@ public extension Image {
     static var rubricSolid: Image { Image("rubricSolid", bundle: .core) }
     static var settingsLine: Image { Image("settingsLine", bundle: .core) }
     static var settingsSolid: Image { Image("settingsSolid", bundle: .core) }
+    static var share: Image { Image("share", bundle: .core) }
```

Comment:
This file is automatically generated by running `yarn build-icons`. These icons are coming from the InstUI repository so we shouldn't modify it manually.

Reply:
Noted!

Reply:
Noted!

Reply:
Noted!